projects/ccf-shared/src/lib/components/organ-selector/organ-selector.component.ts

```diff
@@ -2,22 +2,22 @@ import { AfterViewInit, OnDestroy, Component, EventEmitter, HostBinding, Input,
 import { ResizeSensor } from 'css-element-queries';
 
 export const ALL_ORGANS: OrganInfo[] = [
-  { src: 'app:large_intestine', organ: 'Large Intestine', name: 'Large Intestine', hasSex: true },
-  { src: 'app:heart', organ: 'Heart', name: 'Heart', hasSex: true },
-  { src: 'app:kidney-left', organ: 'Kidney', name: 'Kidney, L', side: 'left', hasSex: true },
-  { src: 'app:kidney-right', organ: 'Kidney', name: 'Kidney, R', side: 'right', hasSex: true },
-  { src: 'app:spleen', organ: 'Spleen', name: 'Spleen', hasSex: true },
-  { src: 'app:bladder', organ: 'Bladder', name: 'Bladder', disabled: true, hasSex: true },
-  { src: 'app:brain', organ: 'Brain', name: 'Brain', disabled: true, hasSex: true },
-  { src: 'app:liver', organ: 'Liver', name: 'Liver', disabled: true, hasSex: true },
-  { src: 'app:lung-left', organ: 'Lung', name: 'Lung, L', disabled: true, side: 'left', hasSex: true },
-  { src: 'app:lung-right', organ: 'Lung', name: 'Lung, R', disabled: true, side: 'right', hasSex: true },
-  { src: 'app:lymph_nodes', organ: 'Lymph Nodes', name: 'Lymph Nodes', disabled: true, hasSex: true },
+  { src: 'app:large_intestine', organ: 'Large Intestine', name: 'Large Intestine', hasSex: true, numResults: 0 },
```

Comment:
Did you test this in the RUI? This seems like it'll show results in there as well... In the business logic in the EUI put this where you are also filtering the data initially: (ie `filter().map(n => { ...n, n.numResults: 0 }`)